config/env/development.js

```diff
@@ -11,10 +9,9 @@
  */
 
 module.exports = {
-  // Feature flags
   featureFlags: {
-    // enable references?
-    reference: true
+    reference: true,
+    i18n: false
```

Comment:
@mrkvon there wasn't any specific reason `i18n` flag was missing from development config, right?

Reply:
Probably not. It was probably supposed to use the default (whatever that was).

Reply:
It was also falsy in at default config but the thing is that we don't deep merge these configs, just shallow merge, so whole `featureFlags`  object here in development config will replace the one in default config.

Reply:
Ah. Ok. Probably good to remember that.

Reply:
Yeah, there were reasons for it (not to have surprise values cropping up from default configs) but it might be surprising.

Also good for clarity to keep the same flags visible on each config so that folks don't need to go opening files or "think abstract" to understand the flag-system. We probably would need them just in production+development and not at all in default.